### PR TITLE
Add Approval Management docs

### PR DESCRIPTION
**Pull request by shuvyA**
image

### CODE REVIEW

File: docs/embeddable-uis/element/approval-management.mdx
Comment (maya-barak) [inline docs/embeddable-uis/element/approval-management.mdx:12]: Need a better introduction here, maybe add link to operation approval doc

**Review by maya-barak — CHANGES_REQUESTED** [review]
more info missing

File: docs/embeddable-uis/element/approval-management.mdx
Comment (maya-barak) [reply docs/embeddable-uis/element/approval-management.mdx:24]: "Approver roles are assigned."- what is the meaning ?

**Review by RazcoDev — APPROVED** [review]

**Review by maya-barak — COMMENTED** [review]
Maybe change "reviewer" role to "Operation Approval Reviewer" or to "_reviewer_"

File: docs/embeddable-uis/element/approval-management.mdx
Comment (maya-barak) [inline docs/embeddable-uis/element/approval-management.mdx:24]: missing explanation about the connection to the operation approval element + add link to the relevant doc (explain that the reviewer role created at the operation element and the developer need to assign the user to this role, also after approve the user who ask the request will assigned to the approver role)

Comment (maya-barak) [reply docs/embeddable-uis/element/approval-management.mdx:24]: missing explanation about the webhook

File: docs/embeddable-uis/element/approval-management.mdx
Comment (maya-barak) [inline docs/embeddable-uis/element/approval-management.mdx:25]: reviewer and approved roles both will be created

File: docs/embeddable-uis/element/approval-management.mdx
Comment (maya-barak) [inline docs/embeddable-uis/element/approval-management.mdx:27]: will be automatically assigned to the approved role

**Review by maya-barak — APPROVED** [review]
LGTM- fix the approve role

**Pull request merged**
